server/www/index.html

```diff
@@ -141,6 +141,12 @@ <h2 ng-bind="site.name"></h2>
                 }(document, 'script'));
             }
         </script>
+        <!--
+        Intercom integration
+        -->
+        <script src="//static.intercomcdn.com/intercom.v1.js"></script>
```

Comment:
Yeah that'd be nice, the only issue is that dynamically loading that kind of JS in angular seems messy but may be someone has a nice wrapper for it.